lib/commands/flutter.js

```diff
@@ -0,0 +1,516 @@
+const fs = require("fs");
+const path = require("path");
+const inquirer = require("inquirer");
+const { teamsCreate } = require("./teams");
+const { projectsCreate, projectsCreatePlatform, projectsListPlatforms } = require("./projects");
+const { sdkForConsole } = require("../sdks");
+const { questionsFlutterConfigure, questionsFlutterSelectPlatforms, questionsFlutterChooseDatabase, questionsFlutterChooseProject } = require("../questions");
+const { success, log, actionRunner, error } = require("../parser");
+const { Command } = require("commander");
+const { globalConfig, localConfig } = require("../config");
+const { databasesList, databasesListCollections } = require("./databases");
+const { sdkForProject } = require("../sdks");
+const { toSnakeCase, toUpperCamelCase } = require("../utils");
+
+const appwriteFile = 'aW1wb3J0ICdwYWNrYWdlOmFwcHdyaXRlL2FwcHdyaXRlLmRhcnQnOwoKZmluYWwgY2xpZW50ID0gQ2xpZW50KCkKICAuc2V0RW5kcG9pbnQoJ3tFTkRQT0lOVH0nKQogIC5zZXRQcm9qZWN0KCd7UFJPSkVDVH0nKTsKCmZpbmFsIGFjY291bnQgPSBBY2NvdW50KGNsaWVudCk7CmZpbmFsIGRhdGFiYXNlcyA9IERhdGFiYXNlcyhjbGllbnQpOwpmaW5hbCBzdG9yYWdlID0gU3RvcmFnZShjbGllbnQpOwoK';
+
+const modelTemplate = 'JUlNUE9SVFMlCmNsYXNzICVOQU1FJSB7CiAgZmluYWwgU3RyaW5nICRpZDsKICBmaW5hbCBTdHJpbmcgJGNyZWF0ZWRBdDsKICBmaW5hbCBTdHJpbmcgJHVwZGF0ZWRBdDsKICBmaW5hbCBTdHJpbmcgJGNvbGxlY3Rpb25JZDsKICBmaW5hbCBTdHJpbmcgJGRhdGFiYXNlSWQ7CiAgZmluYWwgTGlzdDxTdHJpbmc+ICRwZXJtaXNzaW9uczsKCiAgJUFUVFJJQlVURVMlCgogICVOQU1FJSh7CiAgICByZXF1aXJlZCB0aGlzLiRpZCwKICAgIHJlcXVpcmVkIHRoaXMuJGNyZWF0ZWRBdCwKICAgIHJlcXVpcmVkIHRoaXMuJHVwZGF0ZWRBdCwKICAgIHJlcXVpcmVkIHRoaXMuJGNvbGxlY3Rpb25JZCwKICAgIHJlcXVpcmVkIHRoaXMuJHBlcm1pc3Npb25zLAogICAgcmVxdWlyZWQgdGhpcy4kZGF0YWJhc2VJZCwKICAgICVDT05TVFJVQ1RPUl9QQVJBTUVURVJTJQogIH0pOwoKICBmYWN0b3J5ICVOQU1FJS5mcm9tTWFwKE1hcDxTdHJpbmcsIGR5bmFtaWM+IG1hcCkgewogICAgcmV0dXJuICVOQU1FJSgKICAgICAgJGlkOiBtYXBbJ1wkaWQnXSwKICAgICAgJGNyZWF0ZWRBdDogbWFwWydcJGNyZWF0ZWRBdCddLAogICAgICAkdXBkYXRlZEF0OiBtYXBbJ1wkdXBkYXRlZEF0J10sCiAgICAgICRjb2xsZWN0aW9uSWQ6IG1hcFsnXCRjb2xsZWN0aW9uSWQnXSwKICAgICAgJHBlcm1pc3Npb25zOiBMaXN0PFN0cmluZz4uZnJvbShtYXBbJ1wkcGVybWlzc2lvbnMnXSksCiAgICAgICRkYXRhYmFzZUlkOiBtYXBbJ1wkZGF0YWJhc2VJZCddLAogICAgICAlQ09OU1RSVUNUT1JfQVJHVU1FTlRTJQogICAgKTsKICB9CgogIE1hcDxTdHJpbmcsIGR5bmFtaWM+IHRvTWFwKCkgewogICAgcmV0dXJuIHsKICAgICAgJ1wkaWQnOiAkaWQsCiAgICAgICdcJGNyZWF0ZWRBdCc6ICRjcmVhdGVkQXQsCiAgICAgICdcJHVwZGF0ZWRBdCc6ICR1cGRhdGVkQXQsCiAgICAgICdcJGNvbGxlY3Rpb25JZCc6ICRjb2xsZWN0aW9uSWQsCiAgICAgICdcJHBlcm1pc3Npb25zJzogJHBlcm1pc3Npb25zLAogICAgICAnXCRkYXRhYmFzZUlkJzogJGRhdGFiYXNlSWQsCiAgICAgICVNQVBfRklFTERTJQogICAgfTsKICB9CgogIEBvdmVycmlkZQogIFN0cmluZyB0b1N0cmluZygpIHsKICAgIHJldHVybiAnJU5BTUUlICcgKyB0b01hcCgpLnRvU3RyaW5nKCk7CiAgfQp9';
```

Comment:
Is it possible to include the raw templates by themselves to be more transparent?

Reply:
We can do that, I felt this was better than managing multiple files?

Reply:
We can do that, I felt this was better than managing multiple files?
Updated to use files instead.